src/cloudant/design_document.py

```diff
@@ -27,7 +27,7 @@ class DesignDocument(Document):
     :class:`~cloudant.document.Document`.  A DesignDocument object is
     instantiated with a reference to a database and
```

Comment:
The param description for [database](https://github.com/cloudant/python-cloudant/blob/cefd64747f5ea45ef9782f6776f3977edce817ca/src/cloudant/design_document.py#L38-L39) also needs to be fixed. 

Don't forget to add the class name to the param as well as in `:param CouchDatabase database: ...`